promoter/file/s3.go

```diff
@@ -111,28 +117,16 @@ func (p *s3Provider) findRegionForBucket(ctx context.Context, bucket string) (st
 		lookupRegion = "us-east-2"
 	}
 
-	awsConfig := aws.NewConfig()
-	awsConfig = awsConfig.WithRegion(lookupRegion)
-
-	awsSession, err := session.NewSession(awsConfig)
+	// This is an unauthenticated request (it just does a HEAD on the bucket),
+	// so we don't need to be careful about credentials.
+	awsSession, err := session.NewSession()
```

Comment:
Nit: since this is an unauthenticated request, should we enforce using the anonymous credentials (something like we have above)? If I’m not mistaken, this function is going to use credentials if they are available and I’m wondering if we should avoid that.

Reply:
Great point - it's not a risk worth taking :-)  Fixed!